### DISCUSSION TITLE
[Bug]: No sending/receiving possible after changing the device name

### DISCUSSION THREAD

**Discussion by Burny08** [meshtastic/firmware#5060] [Troubleshooting]
### Category

BLE

### Hardware

T-Beam

### Firmware Version

2.5.4

### Description

Hello everyone,
I don't know if it's the firmware or the app. I have long since updated my T-Beam from version 2.2.21 to the current 2.5.5.
The device I use is an iPhone 13 PRO with the latest software.

Now to my problem:
I can set the LoRa settings and then everything works well.
But as soon as I change the name under "User" and set License Operator to on, no messages go out or come in. I have to completely re-flash the T-Beam to be able to use it again. I tested both the current and the oldest stable version. Same problem everywhere.

PS: I tested with 2x T-Beam. Same problem. I also reinstalled the app on iOS. The web client doesn't work via serial either.

Can anyone help? Am I too stupid or is it actually a bug?
Unfortunately, I can't display a debug on the iPhone, or?
Unfortunately, I am only a user, not a programmer.

Thank you so much!

**Comment by fifieldt** [comment]
Setting License Operator disables encryption. So you probably are seeing the lack of messages due to disabling that:

https://meshtastic.org/docs/faq/#amateur-radio-ham
